clients/src/main/java/org/apache/kafka/clients/consumer/internals/Fetcher.java

```diff
@@ -683,8 +685,8 @@ public void onSuccess(ClientResponse response, RequestFuture<Map<TopicPartition,
      * @param listOffsetResponse The response from the server.
      * @param future The future to be completed when the response returns. Note that any partition-level errors will
      *               generally fail the entire future result. The one exception is UNSUPPORTED_FOR_MESSAGE_FORMAT,
-     *               which indicates that the broker does not support the v1 message format and results in a null
-     *               being inserted into the resulting map.
+     *               which indicates that the broker does not support the v1 message format and causes a null value
```

Comment:
Is that right? The comment is describing the `UNSUPPORTED_FOR_MESSAGE_FORMAT` case. It seems that we now we just leave the partition out of the map now?